models/review.go

```diff
@@ -255,3 +255,32 @@ func UpdateReview(r *Review) error {
 	}
 	return nil
 }
+
+// PullReviewersWithType represents the type used to display a review overview
+type PullReviewersWithType struct {
+	User              `xorm:"extends"`
+	Type              ReviewType
+	ReviewUpdatedUnix util.TimeStamp `xorm:"review_updated_unix"`
+}
+
+// GetReviewersByPullID gets all reviewers for a pull request with the statuses
+func GetReviewersByPullID(pullID int64) (issueReviewers map[int64]*PullReviewersWithType, err error) {
+	irs := []*PullReviewersWithType{}
+	err = x.Select("`user`.*, review.type, max(review.updated_unix) as review_updated_unix").
+		Table("review").
+		Join("INNER", "`user`", "review.reviewer_id = `user`.id").
+		Where("review.issue_id = ? AND (review.type = ? OR review.type = ?)", pullID, ReviewTypeApprove, ReviewTypeReject).
+		GroupBy("`user`.id, review.type").
```

Comment:
Why group by `review.type`? I think only the latest review of an user should be shown.

Reply:
Yes, but I need the query fails when using postgresql. See the comment I added in line 277.